### ISSUE TITLE
[TASK] Remove superfluous/extra encryption in keygen

### ISSUE THREAD

**Issue by drewstone** [webb-tools/cggmp-threshold-ecdsa#43]
**Issue summary**
When we originally modified the Zengo mp-ecdsa, we added additional encryption to P2P messages because we were originally broadcasting/gossiping all messages over our network,

Some old commits which added this extra encryption are:
- https://github.com/webb-tools/multi-party-ecdsa/commit/a4895afccc1114d433f7d37f4eb3dec01bc60343
- https://github.com/webb-tools/multi-party-ecdsa/commit/033d79989cf416b8925c28bab95672274eafb28c
- https://github.com/webb-tools/multi-party-ecdsa/commit/cd894b60d41a4cf402bb2b4bde799bf19ed41347
- https://github.com/webb-tools/multi-party-ecdsa/commit/1ef80e1c4a79db3aea34d40297c91c977ed1d7f9

I think we should remove this and make more explicit how this type of message can be encrypted before the P2P messages hit the wire.

**Comment by ivokub** [comment]
Yup, in CGGMP the assumption is that the channels are authenticated between parties and broadcasts are reliable. Our approach is to use Noise protocol on a higher level which provides authenticated encryption between all parties and allows to use untrusted proxy for relaying messages. See https://github.com/mpc-sdk/framework.
